test/functional/CommandCallFunctional.js

```diff
@@ -69,12 +69,11 @@ describe('CommandCall Functional Tests', () => {
         connection.add(new CommandCall({ command: 'system -i wrksyssts', type: 'sh' }));
         connection.run((error, xmlOut) => {
           expect(error).to.equal(null);
-          // xs does not return success property for iSh or iQsh
+          // xs does not return success property for sh or qsh command calls
           // but on error data property = '\n'
           // so lets base success on contents of data.
```

Comment:
Sorry, I meant that it's refereing the data property returned by xml2Json. I suspect that xml2js would return something else instead.

Reply:
You're right, running the sh command test `system -i wrksyssts`  with a typo `ystem -i wrksyssts` yields:

```
<?xml version='1.0'?><myscript><sh error='fast'>
</sh>
</myscript>

{ '$': { error: 'fast' } }

```
There is no inner data for `sh` node and xml2js simply does not define  `myscript.sh[0]._`

tests fails with assertion error though

`Uncaught AssertionError: 
expected undefined to match /(System\sStatus\sInformation)/
`